Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
@MikeMcQuaid I guess that this would also break the currently supported use case of having a formula, which installs different node modules (located in different subfolder in buildpath) to different subfolder in `libexec`. Calling the getter from the first module would pack and install the first module correctly. But every every call of the getter from the second (and follwoing) modules would just install the pack of the first module from the cached path instead of packing and installing the actual second module.

@ilovezfs I guess you have tested it with a module, where the author has explicitly whitelisted the files that should be included in the package using the package.json `files` field. In this case we have luck and `.brew_home` would be included. But it will still be included in packages, which doesn't use the optional `files`field and rely on the default package everying which is not ignored by `.npmignore` behavior.

And regarding the grafana issue: Could you be more specific what issues you are having with it (unfortunately I can't test it myself until at earliest later this evening). Looking at the formuala sources it uses only the local npm installation method and not the 'install-the-module-from-the-current-folder-to-this-prefix' method. Therefore it shouldn't be affected by the npm@5 breaking changes in installation from folder behavior, which now only creates a symlink instead of copying the package over. Also the mixed usage of both `yarn` and `npm` feels a bit inconsistent to me (I suspect the reason for it was that yarn would save on install by default and the motivation of using npm was to not alter the package.json by adding grunt to it. Unfortunaly save on install is now the default for npm@5 too.)